packages/ui/src/Field.tsx

```diff
@@ -13,13 +13,7 @@ import {TextField} from "./TextField";
 export interface FieldProps extends FieldWithLabelsProps {
   name: string;
   label?: string;
-  subLabel?: string;
-  initialValue?: any;
-  handleChange: any;
   height?: number;
-  // Additional validation
-  validate?: (value: any) => boolean;
```

Comment:
I can't find anywhere that actually uses these validate props so I removed them. If there is a spot I missed, let me know!

Reply:
Ah yeah, not sure this actually got used once Field had things like decimals implemented. 👍